### ISSUE TITLE
Fatal Exception: java.lang.IllegalStateException: Already resumed

### ISSUE THREAD

**Issue by TemMax** [apphud/ApphudSDK-Android#47]
Hi. Continuation#resume calls twice in your code at [here](https://github.com/apphud/ApphudSDK-Android/blob/68e172ed6f70ac35d2c070cf4cf3397c13fe638b/sdk/src/main/java/com/apphud/sdk/internal/SkuDetailsWrapper.kt#L118)

It could be caused by different versions of coroutines. AppHud sdk has 1.5.0 and 1.6.0 in my project.

Full trace from Firebase crashlytics:

```
Fatal Exception: java.lang.IllegalStateException: Already resumed
       at kotlin.coroutines.SafeContinuation.resumeWith(SafeContinuationJvm.kt:44)
       at com.apphud.sdk.internal.SkuDetailsWrapper$queryAsyncEx$$inlined$suspendCoroutine$lambda$1$1.onSkuDetailsResponse(SkuDetailsWrapper.kt:118)
       at com.android.billingclient.api.zzv.run(com.android.billingclient:billing@@4.0.0:1)
       at com.android.billingclient.api.zzw.run(com.android.billingclient:billing@@4.0.0:4)
       at android.os.Handler.handleCallback(Handler.java:809)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:166)
       at android.app.ActivityThread.main(ActivityThread.java:7555)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:469)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:963)
```

**Comment by vladimir-slatvinskiy** [comment]
fixed v1.5.6

**Comment by TemMax** [comment]
This is one more place where crash happens. You are remember to fix it here :)
```
Fatal Exception: java.lang.IllegalStateException: Already resumed
       at kotlin.coroutines.SafeContinuation.resumeWith(SafeContinuationJvm.kt:44)
       at com.apphud.sdk.internal.BillingWrapper$connect$2$1.onBillingSetupFinished(BillingWrapper.kt:54)
       at com.android.billingclient.api.zzaf.zzd(com.android.billingclient:billing@@4.0.0:1)
       at com.android.billingclient.api.zzaf.zzb(com.android.billingclient:billing@@4.0.0:3)
       at com.android.billingclient.api.zzad.run(com.android.billingclient:billing@@4.0.0:2)
       at com.android.billingclient.api.zzw.run(com.android.billingclient:billing@@4.0.0:4)
       at android.os.Handler.handleCallback(Handler.java:888)
       at android.os.Handler.dispatchMessage(Handler.java:100)
       at android.os.Looper.loop(Looper.java:213)
       at android.app.ActivityThread.main(ActivityThread.java:8178)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:513)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1101)
```

**Comment by ren6** [comment]
@TemMax this has been fixed in 1.5.8. Thank you for pointing out!
